src/utils/utils.go

```diff
@@ -0,0 +1,20 @@
+package utils
+
+import (
+	"fmt"
+
+	"golang.org/x/crypto/bcrypt"
+)
+
+func HashPassword(password string) (string, error) {
+	hashedPassword, err := bcrypt.GenerateFromPassword([]byte(password), bcrypt.DefaultCost)
+
+	if err != nil {
+		return "", fmt.Errorf("could not hash password %w", err)
```

Comment:
this call should not fail (under any normal operation)
Can you do a `lo.Must` wrapper simply instead ?